Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -974,6 +981,8 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
                 handleMouseDraggedOffTrackedAnnotation();
                 return true;
             }
+
+            continueTrackingSelection(*pageIndex, pointInPageSpace);
```

Comment:
Is there a reason `continueTrackingSelection` implies that the `UnifiedPDFPlugin` could not handle the mouse event for some reason?

Reply:
No, no, good point. I'll fix the mouse event handler(s) return values.

Reply:
Returning `true` following [53cae5f](https://github.com/WebKit/WebKit/commit/53cae5fe11eca85d9ea822cdec0181e4a4f4dcf6)